src/constants/index.ts

```diff
@@ -0,0 +1,5 @@
+const NEW_OBJECT_TAG = 'new';
+
+export {
+    NEW_OBJECT_TAG,
+};
```

Comment:
Why was this added? Remove.

At our code we can use new- prefix globally for all new objects that haven't been saved. If it needs documenting, cotributions.md is the right place.

Reply:
I don't want us to have magic strings that we use for matching, that is only declared in contributions.md. We need to match this prefix to know which ones are added and which ones are old, and we can't start writing hard-coded prefixes.